exchange/price_granularity.go

```diff
@@ -12,7 +12,7 @@ func GetPriceBucket(cpm float64, config openrtb_ext.PriceGranularity) string {
 	cpmStr := ""
 	bucketMax := 0.0
 	increment := 0.0
-	precision := config.Precision
+	precision := *config.Precision
```

Comment:
Is this potentially dangerous? Yes. But it's ok due to app design and full test coverage. 🤞 Changing this to return an error caused an annoying domino effect for a case which will never occur due to protections run earlier in the flow.